src/Elastic.OpenTelemetry/Diagnostics/LoggingEventListener.cs

```diff
@@ -34,7 +34,7 @@ public LoggingEventListener(ILogger logger)
 	{
 		_logger = logger;
 
-		var eventLevel = AgentLoggingHelpers.GetElasticOtelLogLevel();
+		var eventLevel = AgentLoggingHelpers.GetElasticOtelLogLevelFromEnvironmentVariables();
```

Comment:
Does this only listen to env variables ignoring ``Elastic:OpenTelemetry:FileLogLevel`` from IOptions?

Reply:
Good catch! Yeah, this should be updated to use the configured level. In fact, it should be the minimum of either that or the logging configuration section for "Elastic.OpenTelemetry" since we want either logger to get the events they opt into.